bokehjs/src/coffee/document.coffee

```diff
@@ -102,21 +102,24 @@ class Document
   solver: () ->
     @_solver
 
-  resize: (width=null, height=null) ->
+  resize: (jquery_event, width=null, height=null) ->
```

Comment:
@mattpap this PR was already merged, with the expectation to fix it up. Can you create and merge a PR for this immediately? @canavandl was also up very late on this and I'd like to get it in sooner than later.